k8s/jiva/README.md

```diff
@@ -0,0 +1,124 @@
+This document will help to delete the auto-generated snapshots created when a Jiva replica restart happened or new replica got added to the Jiva controller. The steps for deleting the auto-generated snapshots  are the following:
```

Comment:
This document will help to delete the auto-generated snapshots created during a Jiva replica restart or when a new replica is added to the Jiva controller. The steps for deleting the auto-generated snapshots are the following